docs/sphinx_source/c_namespaces/fmt.rst

```diff
@@ -0,0 +1,6 @@
+Namespace: Fmt
+==============
+.. doxygennamespace:: fmt
+   :members:
+   :undoc-members:
+   :private-members:
```

Comment:
Can you remove this fmt.rst file again. It is a weird remnant that is produced and is not needed.